satpy/scene.py

```diff
@@ -34,6 +34,7 @@
 from satpy.node import CompositorNode, MissingDependencies, ReaderNode
 from satpy.readers import load_readers
 from satpy.resample import get_area_def, prepare_resampler, resample_dataset
+from satpy.utils import _get_storage_options_from_reader_kwargs, convert_remote_files_to_fsspec
```

Comment:
I think a couple of us talked about this a little while ago. An underscore function should only be used inside its own module. An underscore module can be used inside the package, but its functions should still be public (no underscore) if they are used outside the module.

Reply:
Changed `get_storage_options_from_reader_kwargs()` to public function.